microsoft-edge/devtools-guide-chromium/whats-new/2023/02/devtools-110.md

```diff
@@ -0,0 +1,175 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/09/2023
+---
+# What's New in DevTools (Microsoft Edge 110)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Taking heap snapshots in the Memory tool is now faster
+
+<!-- Subtitle: When taking a heap snapshot greater than 1 GB in size, the time spent generating the snapshot in the Memory tool is at least 70% faster. -->
+
+In previous versions of Microsoft Edge, generating large heap snapshots (over 1 GB) in the **Memory** tool could take anywhere from 2 to 15 minutes, depending on the size of your website or app, and how many elements were stored in memory.
+
+In Microsoft Edge 110, the **Memory** tool is now much faster at taking heap snapshots.  Taking a heap snapshot is now around 70% to 86% faster.  For larger heap snapshots, we're seeing the greatest improvement in speed.  If you have noticed that taking heap snapshots is much faster, please leave us feedback in [[Feedback] Taking heap snapshots in the Memory tool is now faster #131](https://github.com/MicrosoftEdge/DevTools/issues/131).
+
+![Taking a heap snapshot in the Memory tool](./devtools-110-images/faster-heap-snapshot.png)
+
+See also:
+* [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md#take-a-snapshot)
+
+
+<!-- ====================================================================== -->
+## Move tools between the Activity Bar and Quick View more easily in Focus Mode
+
+<!-- Subtitle: Customize both the Activity Bar and Quick View in Focus Mode to see only the tools you care about. -->
+
+Microsoft Edge 110 contains various improvements to **Focus Mode** in DevTools.  The following changes make it easier to customize the Activity Bar and Quick View, to show only the tools that you're currently interested in.
+
+
+<!-- ------------------------------ -->
+#### Right-click menu shows the current location of the other toolbar to move a tool to
+
+The right-click menu for moving an optional tool between Quick View and the Activity Bar now includes the position of the Activity Bar, such as:
+* **Move to bottom Quick View**
+* **Move to side Quick View**
+* **Move to left Activity Bar**
+* **Move to top Activity Bar**
+
+![Right-click menu showing 'Move to bottom Quick View'](./devtools-110-images/move-to-bottom-quick-view.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ------------------------------ -->
+#### Tool name remains visible when you right-click the tool icon
```

Comment:
```suggestion
#### The right-click menu no longer displays over the tool in the toolbar
```
We've already called it the "right-click menu" in the previous announcement. I don't have a preference for a better name (maybe "context menu") but I just want us to be consistent between this announcement and the previous one

Reply:
Changed locally.

fwiw, Bing hit count:
"right-click menu": 5.5M hits
"context menu": 2M hits
"shortcut menu": 1.4M hits

"right-click" is popular in common use b/c concrete, not abstract.

__

per previous grammar research, going with:
#### The right-click menu is no longer displayed over the tool in the toolbar
Usage is either "x is displayed", or "x displays y".
Corp style guide removed the entry for "display, appear".